### PR TITLE
Add secondary bars

### PR DESCRIPTION
**Pull request by attila-at-hyper**
The initial development went (unintentionally) to the “master” branch. Please have a look at the entire source, not just the changes of this PR.

### CODE REVIEW

**Comment by attila-at-hyper** [comment]
This is how it looks like in action:

ios simulator screen shot mar 20 2015 11 24 23 am

File: Source/BarChartView.m [hunk 1]
```diff
@@ -9,6 +9,8 @@
 static CGFloat BarChartMinimumAndMaximumValueLabelWidth = 60.0f;
```
Comment (kostiakoval) [inline Source/BarChartView.m]: maybe it should be `static const CGFloat` - so you can't change it
@attila-at-hyper

**Comment by attila-at-hyper** [comment]
@kostiakoval: added "const" per your suggestion

**Comment by kostiakoval** [comment]
Thanks! It was also mentioned by elvis in the playbook.

**Pull request merged**